Created Feedback page and Thankyou page

DinalPrabathiya opened the pull request:
Added a Feedback page for the users to add their feedback and their requests with a success message after submitting through a different Thank_you page. Modified the footer to include the feedback button where the users can access the feedback page.

s-bonnor commented on home/templates/accounts/sign-up.html:
This looks like its been included by accident

DinalPrabathiya replied:
I have added the .vs files cause i accidently reset my whole GitHub progress when i was going to commit and i had to update all files to how it was before the reset. that is why it is showing as updated. what can i do about this?

s-bonnor reviewed (changes requested):
Please update your pull request to not add the .vs folder and files under it

DinalPrabathiya commented:
I will update this within the week. I am currently in Canberra representing Deakin for Uni Nationals.
Thanks


Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: s-bonnor ***@***.***>
Sent: Saturday, September 7, 2024 9:35:57 AM
To: Hardhat-Enterprises/website ***@***.***>
Cc: DINAL JALITH PRABATHIYA HATHTHOTUWA GAMAGE ***@***.***>; Author ***@***.***>
Subject: Re: [Hardhat-Enterprises/website] Created Feedback page and Thankyou page (PR #125)


@s-bonnor requested changes on this pull request.

Please update your pull request to not add the .vs folder and files under it

________________________________

On home/templates/accounts/sign-up.html<https://github.com/Hardhat-Enterprises/website/pull/125#discussion_r1747814611>:

This looks like its been included by accident

—
Reply to this email directly, view it on GitHub<https://github.com/Hardhat-Enterprises/website/pull/125#pullrequestreview-2287274086>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A7RYPQVQODJAYQGEGOBOUHLZVI353AVCNFSM6AAAAABNVMXL4OVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDEOBXGI3TIMBYGY>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

DinalPrabathiya commented:
Hey Sam, i am not sure what has gone wrong in that file could you please help me out here as i am not sure what i have added wrong to that file as i updated it to my needs as well but just the original content to get to my needs.
